src/Misc/layoutbin/powershell/Add-AzurePowerShellCapabilities.ps1

```diff
@@ -89,7 +88,8 @@ function Get-FromSdkPath {
 }
 
 Write-Host "Env:PSModulePath: '$env:PSModulePath'"
-$null = (Get-FromModulePath -Classic:$false) -or
+$null = (Get-FromModulePath -ModuleName:"Az.Profile") -or
```

Comment:
Unfortunately "Get-Module -Name Az -ListAvailable" is not working because of the  [PowerShell bug](https://github.com/PowerShell/PowerShell/issues/8125).

I Choose Az.Profile because Azure.Profile module has been used in both SDK path flow and Module path flow.

Reply:
Sweet bug. that's why i hate rename. :)
So we are not really reporting you have `Az` installed on the machine, we report the capability as long as you have the `Az.Profile` installed.

Reply:
Yes. That is true.
As long as Az.Profile module is there, Initialize-Azure step will work. Task script can fail saying command not found, if corresponding module is not present.
We have similar behaviour in the SDK path flow as well.

Reply:
understand, i am fine with the changes since i can't see any reasonable workaround here.
leave a comment in the code about why we have to search `Az.Profile` instead of `Az`, and keep in mind to re-visit the module name after they release as they might rename the module. :)

Reply:
Added comment.
I will keep an eye of Az module version 1.0 release for Az.Profile name.